sql/moz-fx-data-shared-prod/apple_ads_derived/ad_group_report_v1/query.sql

```diff
@@ -0,0 +1,4 @@
+SELECT
+  *
```

Comment:
I'd recommend explicitly listing the columns to be selected in these mirroring ETLs, since by default Fivetran will automatically add new columns to its tables (`SELECT *` in the downstream views is fine).